frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/ticketGroups/groups/groupRulesForm/groupRulesForm.helpers.ts

```diff
@@ -52,18 +53,18 @@ export const formRuleToGroupRule = ({ values, field, ...rule }: IFormRule): IGro
 	...rule,
 	field: {
 		operator: field.operator,
-		values: field.values.map((v) => v.value),
+		values: mapFormArrayToArray(field.values),
 	},
-	values: (values || []).map((v) => v.value),
+	values: mapFormArrayToArray(values || []),
 });
 
 export const groupRuleToFormRule = ({ values, field, ...rule }: IGroupRule): IFormRule => ({
 	...rule,
 	field: {
 		operator: field.operator,
-		values: (field.values || []).map((value) => ({ value })),
+		values: mapArrayToFormArray(field.values || []),
```

Comment:
maybe `mapArrayToFormArray` can cater for undefined or null values and treat them as []. You  reckon there could be a problem with that?